resources/js/components/module/templates/shared/VideoPlayback.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="video-template justify-content-center mb-5 row">
+    <div class="video-template__video col-12 col-md-10">
```

Comment:
Add a `p-0` so the video and take the full on mobile view.

Reply:
Resolved